README.md

```
@@ -86,6 +86,96 @@ npm test
+
+### 起動
+
+```bash
+docker compose up --build
+```
+
+起動後、以下のURLでアクセスできます:
+
+| サービス         | URL                       |
+| ---------------- | ------------------------- |
+| アプリケーション | http://localhost          |
```

markdownlint-cli2 (0.20.0)

[warning] Bare URL used

(MD034, no-bare-urls)